### ISSUE TITLE
Verified DM should include the server name

### ISSUE THREAD

**Issue by sahil-kale** [sahil-kale/tron-discord-bot#260]

**Comment by nlao1** [comment]
Hi, I can take this one.

**Comment by sahil-kale** [comment]
Sweet, it's yours

**Comment by sahil-kale** [comment]
@nlao1 if you look in this pull request (https://github.com/sahil-kale/tron-discord-bot/pull/249/files), there is code that did something similar on Kick/Ban. Hope it helps! Feel free to reply in thread if you've got questions :)

**Comment by nlao1** [comment]
Is there a way to test my code with the bot functionality?

**Comment by sahil-kale** [comment]
Yup! So, you can create a discord bot token yourself (Link here: https://www.freecodecamp.org/news/create-a-discord-bot-with-python/). Typically with open-source bots, you can run them with your own tokens so that you don't inadvertently log in with the main bot and mess it up. The steps below sound tedious, but it shouldn't take more than 15 minutes :)

Once you have the token, make a file with the name of ".env" in the root directory. 
The key for the token is BOT_TOKEN

Additionally, you'll need to make a key called BOT_OWNERS. Set it to your discord tag in a string.

You'll also need to spin up a mongo server, @AadityaChaudhary can link a quick tutorial. 

At the end, your ".env" file should look like this:
BOT_TOKEN=token_goes_here_notice_the_lack_of_quotation_marks
BOT_OWNERS="yourdiscordIDhere_notice_that_there_are_quotation_marks"
MONGO_CONNECTION_STRING=mongodb+srv://mongoconnectionstuff

**Comment by sahil-kale** [comment]
I'll defer to my colleagues in getting a quick dev environment set up. You might be the reason we end up using docker or something :P

**Comment by nlao1** [comment]
This is very late, but I'm getting an error that has to do with the line
os.environ['UW_API_KEY'] = os.getenv('UW_API_KEY')

Any ideas?

**Comment by zaydtahir** [comment]
Thats because we used to access UW API to do some verification stuff about programs, but that is no longer used. That should be removed now or soon so you shouldn't run into the issue. Sorry for taking so long to respond.
